app_dart/lib/src/foundation/utils.dart

```diff
@@ -200,10 +200,10 @@ Future<void> insertBigquery(String tableName, Map<String, dynamic> data, Tableda
 }
 
 /// Validate test ownership defined in [testOwnersContent] for tests configured in `ciYamlContent`.
-List<String> validateOwnership(String ciYamlContent, String testOwnersContent) {
+Future<List<String>> validateOwnership(String ciYamlContent, String testOwnersContent) async {
   final List<String> noOwnerBuilders = <String>[];
   final YamlMap? ciYaml = loadYaml(ciYamlContent) as YamlMap?;
-  final pb.SchedulerConfig schedulerConfig = schedulerConfigFromYaml(ciYaml);
+  final pb.SchedulerConfig schedulerConfig = await schedulerConfigFromYaml(ciYaml, ensureBringupTargets: false);
```

Comment:
Since there's only one case where `ensureBringupTargets = true`, we should set the default value to false.